### PR TITLE
Danish translation 

### PR DESCRIPTION
**Pull request by caltv93**
Danish translation

### CODE REVIEW

**Review by decryp2kanon — CHANGES_REQUESTED** [review]
pls remove english correction at this time. let's do one by one. do not together.

**Review by cryptozeny — APPROVED** [review]
thanks!

File: da.json [hunk 1]
```diff
@@ -0,0 +1,198 @@
+{
+	// Address book screen
+	"addressBook.title": "Adressebog",
+	"addressBook.sendButton": "Send",
```
Comment (decryp2kanon) [inline da.json:4]: send is send in danish?

File: da.json [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+	"addressBook.addressInput": "Adresse",
+	"addressBook.addAddressButton": "Tilføj adresse",
+	"addressBook.empty": "Adressebogen er tom.",
+
+	// Adding an address screen
+	"addAddress.title": "Tilføjer en adresse",
+	"addAddress.subtitle": "Føj adressen til adressebogen",
+	"addAddress.existNameAlert": "En adresse med dette navn findes allerede. Prøv igen!",
+	"addAddress.existAddressAlert": "En adresse findes allerede i adressebogen. Prøv igen!",
+	"addAddress.successAlert": "Succes!",
```
Comment (decryp2kanon) [inline da.json:16]: in danish?

File: da.json [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+	"transactionDetails.dateTooltipText": "Dato",
+	"transactionDetails.confirmationsTooltipText": "bekræftelser",
+	"transactionDetails.txidTooltipText": "TXID",
+	"transactionDetails.fromTooltipText": "Fra",
+	"transactionDetails.toTooltipText": "Til",
+	"transactionDetails.explorerButton": "Blockchain explorer",
+	"transactionDetails.closeButton": "Luk",
+	"transactionDetails.amountText": "Beløb",
+	"transactionDetails.feeText": "Afgift",
+	"transactionDetails.alertTitle": "Send",
```
Comment (decryp2kanon) [inline da.json:151]: danish?

File: da.json [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+	// Send screen
+	"send.title": "Send",
+	"send.transactionSubtitle": "Overførselsdetaljer",
+	"send.fromText": "Fra",
+	"send.toText": "Til",
+	"send.feeText": "Afgift",
+	"send.amountText": "Beløb",
+	"send.addressExampleInput": "sugar1qcu07vm47ceh7lec3umx7nv7e8tm8r8zw4l2uyx",
+	"send.confirmButton": "Bekræft",
+	"send.successAlert": "Succes!",
```
Comment (decryp2kanon) [inline da.json:119]: danish?

File: da.json [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+	"importKey.keyInput": "WIF nøgle",
+	"importKey.keyTooltipText": "Skal være en gyldig nøgle",
+	"importKey.addButton": "Tilføj",
+	"importKey.cancelButton": "Afbestil",
+	"importKey.successAlert": "Succes!",
+	"importKey.confirmAlertButton": "OK",
+	"importKey.importErrorAlert": "Forkert WIF-nøgle. Prøv igen!",
+
+	// Send screen
+	"send.title": "Send",
```
Comment (decryp2kanon) [inline da.json:111]: danish?

**Review by decryp2kanon — APPROVED** [review]

**Comment by decryp2kanon** [comment]
@caltv93 pls add your sugar address here!

**Comment by caltv93** [comment]
sugar1qe2lpuzz72u864e86gfjcmlsalmf3j3d5qe7vv0

**Comment by decryp2kanon** [comment]
@AestheticSenpai will reward soon

thanks!

**Pull request merged**